README.md

```diff
@@ -96,7 +96,7 @@ Initialization
 var preview = new Box.Preview();
 preview.show(fileId, accessToken, { options });
 ```
-Where `fileId` is a string `Box_File` id and `accessToken` is a Box API access token.
+Where `fileId` is a string or number `Box_File` id and `accessToken` is a Box API access token.
```

Comment:
Let's not change the documentation. We can add some more error recovery but we should recommend devs to still send strings.